addon/bootstrap.js

```diff
@@ -98,18 +98,20 @@ async function startup(addonData, reason) {
   }
   await studyUtils.startup({reason});
 
-  log.debug(`info ${JSON.stringify(studyUtils.info())}`);
+  console.log(`info ${JSON.stringify(studyUtils.info())}`);
 
-  let value = HomepageStudy.reportValue(studyUtils.studyName);
+  let value = HomepageStudy.reportValue(studyConfig.studyName, studyConfig.isSimulation, studyConfig.rapporPath);
```

Comment:
What about calling `reportValue` with the simulation argument and then simply using the returned data to power the simulation specific functions from `Simulator.jsm`?